pyaedt/circuit.py

```diff
@@ -1962,7 +1964,111 @@ def create_ami_schematic_from_snp(
              Whether to automatically assign differential pairs. The default is ``False``.
         design_name : str, optional
             New schematic name. The default is ``"LNA"``.
+        ibis_rx_file : str, optional
+            Ibis receiver file.
+        create_setup : bool, optional
+            Whether if create transient or ami setup or not. The default is ``True``.
+
+        Returns
+        -------
+        (bool, list, list)
+            First argument is ``True`` if successful.
+            Second and third arguments are respectively the names of the tx and rx mode probes.
+        """
 
+        return self.create_ibis_schematic_from_snp(
+            input_file=input_file,
+            ibis_tx_file=ibis_ami,
+            component_name=component_name,
+            tx_buffer_name=tx_buffer_name,
+            rx_buffer_name=rx_buffer_name,
+            tx_pins=tx_pins,
+            tx_refs=tx_refs,
+            rx_pins=rx_pins,
+            rx_refs=rx_refs,
+            use_ibis_buffer=use_ibis_buffer,
+            differential=differential,
+            bit_pattern=bit_pattern,
+            unit_interval=unit_interval,
+            use_convolution=use_convolution,
+            analyze=analyze,
+            design_name=design_name,
+            is_ami=True,
+            ibis_rx_file=ibis_rx_file,
+            create_setup=create_setup,
+        )
+
+    @pyaedt_function_handler()
+    def create_ibis_schematic_from_snp(
+        self,
+        input_file,
+        ibis_tx_file,
+        tx_buffer_name,
+        rx_buffer_name,
+        tx_pins,
+        rx_pins,
+        ibis_rx_file=None,
+        tx_refs=None,
+        rx_refs=None,
+        component_name=None,
+        use_ibis_buffer=True,
+        differential=True,
+        bit_pattern=None,
+        unit_interval=None,
+        use_convolution=True,
+        analyze=False,
+        design_name="IBIS",
+        is_ami=False,
+        create_setup=True,
+    ):
+        """Create a schematic from a Touchstone file and automatically set up an IBIS-AMI analysis.
+
+        Parameters
+        ----------
+        input_file : str
+            Full path to the sNp file.
+        ibis_tx_file : str
+            Full path to the IBIS file.
+        tx_buffer_name : str
+            Transmission buffer name. It can be a buffer or a ibis pin name.
+            In this last case the user has to provide also the component_name.
+        rx_buffer_name : str
+            Receiver buffer name.
+        tx_pins : list
+            Pins to assign the transmitter IBIS.
```

Comment:
```suggestion
            Pins to assign to the transmitter IBIS.
```
@maxcapodi78 are this pins to assign to the transmitter?